compiler/graindoc/docblock.re

```diff
@@ -126,10 +126,17 @@ let () =
       Some(msg);
     | MissingUnlabeledParamType({idx}) =>
       let msg =
-        Printf.sprintf(
-          "Unable to find a type for parameter at index %d. Make sure a parameter exists at this index in the parameter list.",
-          idx,
-        );
+        if (idx == 0) {
+          Printf.sprintf(
+            "Unable to find a type for parameter at index %d. Please note that parameter count starts at 1.",
```

Comment:
If we're starting it from 1 I think we should do away with the wording "index" as that almost unanimously implies "starting from 0" in programming languages. Maybe "position"?

Reply:
Thoughts on just saying `Unable to find a type for parameter %d`?

Reply:
Sure sounds good to me